### ISSUE TITLE
学習プログラムを安定させる

### ISSUE THREAD

**Issue by DenDen047** [DenDen047/DeepRobotics#34]
- [x] train.pyのdone判定を修正
- [x] tensorboardについて詳しく調べる
- [ ] 細かいwarningなどをなくす

**Comment by DenDen047** [comment]
`Unexpected end of /proc/mounts line`は無視する
https://devtalk.nvidia.com/default/topic/1027077/container-pytorch/-quot-unexpected-end-of-proc-mounts-line-overlay-quot-on-p3-8xlarge/

**Comment by DenDen047** [comment]
`UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.`は必要なメッセージなので無視
https://qiita.com/hachisukansw/items/2d85c3d2c021f9a7e703

**Comment by DenDen047** [comment]
`WARN: Trying to monitor an environment which has no 'spec' set. This usually means you did not create it via 'gym.make', and is recommended only for advanced users.`
これも無視

**Comment by DenDen047** [comment]
拡張命令セットでTensorFlowをコンパイルする方法
https://qiita.com/yijyun-wang/items/12a76667482551eadabc

**Comment by DenDen047** [comment]
Bazelのインストール
https://docs.bazel.build/versions/master/install-ubuntu.html

**Comment by DenDen047** [comment]
https://www.kabuku.co.jp/developers/tensorflow_source_build

**Comment by DenDen047** [comment]
GPUがあるので，` Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA`は無視する

**Comment by DenDen047** [comment]
https://github.com/tensorflow/tensorflow/issues/9512

**Comment by DenDen047** [comment]
https://github.com/openai/baselines/blob/master/baselines/common/tf_util.py

**Comment by DenDen047** [comment]
前回の結果をそのまま使うと動きが収束して，止まってしまう

**Comment by DenDen047** [comment]
`logstd`に問題があるから，`std`が小さくなるように修正する必要がある

**Comment by DenDen047** [comment]
reward を　https://s3-us-west-1.amazonaws.com/disneyresearch/wp-content/uploads/20180625141830/Automated-Deep-Reinforcement-Learning-Environment-for-Hardware-of-a-Modular-Legged-Robot-Paper.pdf　に合わせる

**Comment by DenDen047** [comment]
https://www.google.co.jp/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=2ahUKEwjc4um44-TdAhUCa7wKHdepDnkQjRx6BAgBEAU&url=https%3A%2F%2Fwww.researchgate.net%2Ffigure%2FAverage-roll-pitch-and-yaw-angles_fig2_262055313&psig=AOvVaw2Mh7yUpw7dFRckjWhwopJ9&ust=1538467294418595

**Comment by DenDen047** [comment]
モータの角度を変更しても，Simulationに反映されない問題

**Comment by DenDen047** [comment]
`action_low =action_high = [0.] * self.num_joints` のように変数宣言すると，アドレス参照になってしまい，`action_low`と`action_high`の値が共有されていたことが原因

**Comment by DenDen047** [comment]
observationの観測値を正規化した方が良さそう
